frontend/src/views/CellCards/components/CellCard/components/MarkerGeneTables/index.tsx

```diff
@@ -163,35 +163,70 @@ const MarkerGeneTables = ({ cellTypeId }: Props) => {
       if (!genes) return [];
       const rows = [];
 
+      const publicationTitlesToIndex = new Map();
+      let index = 0;
       for (const markerGene of genes) {
         if (markerGene.tissue_general !== selectedOrgan) continue;
-        // multiple publications for a single gene are joined by ";;"
-        const publications = markerGene.publication.split(";;");
         const publicationTitles = markerGene.publication_titles.split(";;");
+        for (let i = 0; i < publicationTitles.length; i += 1) {
+          if (
+            !publicationTitlesToIndex.has(publicationTitles[i]) &&
+            publicationTitles[i] !== ""
+          ) {
+            publicationTitlesToIndex.set(publicationTitles[i], index);
+            index += 1;
+          }
+        }
+      }
+      for (const markerGene of genes) {
+        if (markerGene.tissue_general !== selectedOrgan) continue;
+        // multiple publications for a single gene are joined by ";;"
+        const publications = Array.from(
+          new Set(markerGene.publication.split(";;"))
+        );
+        const publicationTitles = Array.from(
+          new Set(markerGene.publication_titles.split(";;"))
+        );
         const publicationLinks = (
           <PublicationLinkWrapper>
-            {publications.map((publication, index) => {
-              if (publication && publicationTitles[index]) {
+            {publicationTitles
+              .sort((a, b) => {
                 return (
-                  <Tooltip
-                    key={`${publication}-${index}-tooltip`}
-                    placement="right"
-                    width="default"
-                    arrow={false}
-                    title={publicationTitles[index]}
-                    leaveDelay={0}
-                  >
-                    <span key={`${publication}-${index}-span`}>
-                      <Link
-                        key={`${publication}-${index}`}
-                        label={`[${index + 1}]`}
-                        url={`https://doi.org/${publication}`}
-                      />
-                    </span>
-                  </Tooltip>
+                  publicationTitlesToIndex.get(a) -
+                  publicationTitlesToIndex.get(b)
                 );
-              }
-            })}
+              })
+              .map((publicationTitle, index) => {
+                if (publicationTitle && publications[index]) {
+                  const referenceIndexLabel =
+                    publicationTitlesToIndex.get(publicationTitle) + 1;
+                  return (
+                    <Tooltip
+                      key={`${publications[index]}-${index}-tooltip`}
+                      placement="top"
+                      width="default"
+                      arrow={false}
+                      title={
+                        <div>
+                          {publicationTitle.split("\n\n").at(0)}
```

Comment:
formatting here to put the citation (delimited by `\n\n` in the fixture) in italics.